src/ui/buttons/buttonTextIcone/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { Button, Flex, HStack, Text } from "@chakra-ui/react";
+import { ButtonTextIconeProps } from "./props";
+
+
+const ButtonTextIcone = ({
+  text,
+  textFont,
+  textcolor,
+  border,
+  padding,
+  borderRadius,
+  startEnhancer,
+  endEnhancer,
+  marginRightLeftIcon,
+  marginRightRightIcon,
+  bg,
+  variant,
+  width,
+  height,
+  colorButton,
+}: ButtonTextIconeProps) => {
+  return (
+    <Button
+      flexDirection='row'
+      alignItems='center'
+      padding={padding || "12px"}
```

Comment:
rather than doing this padding={padding || "12px"} u can give padding a default value in the prop and then just do this padding={padding}